sql-statements/sql-statement-alter-resource-group.md

```diff
@@ -79,18 +104,19 @@ Query OK, 0 rows affected (0.08 sec)
 
 ```sql
 SELECT * FROM information_schema.resource_groups WHERE NAME ='rg1';
-+------+------------+----------+-----------+
-| NAME | RU_PER_SEC | PRIORITY | BURSTABLE |
-+------+------------+----------+-----------+
-| rg1  |       100  | MEDIUM   | YES       |
-+------+------------+----------+-----------+
++------+------------+----------+-----------+-------------+
+| NAME | RU_PER_SEC | PRIORITY | BURSTABLE | QUERY_LIMIT |
++------+------------+----------+-----------+-------------+
+| rg1  | 100        | MEDIUM   | YES       | NULL        |
++------+------------+----------+-----------+-------------+
 1 rows in set (1.30 sec)
 ```
 
 ```sql
 ALTER RESOURCE GROUP rg1
   RU_PER_SEC = 200
   PRIORITY = LOW;
+  QUERY_LIMIT = (EXEC_ELAPSED='1s' ACTION=COOLDOWN WATCH=EXACT[30s]);
```

Comment:
```suggestion
  QUERY_LIMIT = (EXEC_ELAPSED='1s' ACTION=COOLDOWN WATCH=EXACT DURATION '30s');
```

Reply:
`(EXEC_ELAPSED='1s' ACTION=COOLDOWN WATCH=EXACT DURATION='30s')` all fields use the same format is better